src/main/java/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> storage = new HashMap<>();

```

Comment:
Create `Fruit` class with field `String name` and use it instead of String in your aplication logic